### DISCUSSION TITLE
self-extinguishing

### DISCUSSION THREAD

**Discussion by h7899** [firemodels/fds#13133] [Q&A]
I want to try to simulate the self-extinguishing phenomenon, but there is a "Minimum density" situation, can you give me a little advice, thank you very much.

**Comment by mcgratta** [comment]
We can provide no advice based on this short message. You need to provide more information.

**Comment by drjfloyd** [comment]
See 21.3 Pressure Considerations in Long Tunnels

**Reply by h7899** [reply]
FOR THIS PROBLEM I SET &PRES TUNNEL_PRECONDITIONER=T, MAX_PRESSURE_ITERATIONS=50 / AND &PRES TUNNEL_PRECONDITIONER=T, MAX_PRESSURE_ITERATIONS=100 U/, STILL CAN'T SOLVE THE PROBLEM, DO YOU HAVE ANY OTHER SUGGESTIONS, THANK YOU.

**Comment by rmcdermo** [comment]
I am running the case.  So far, a few seconds in, everything is fine.  If you get a minimum density warning, it might be just a warning.  Are you also getting an instability?  Please help us understand what your problem is.  It is very hard to guess and takes us a lot of time to analyze everyone's issues.

**Reply by h7899** [reply]
When the minimum density problem occurs, the value is unstable and cannot be calculated

**Reply by drjfloyd** [reply]
You have this input:
&COMB RADIATIVE_FRACTION=0.456 /

The input of RADIATIVE_FRACTION is not on COMB in 6.9.1. It was moved to REAC in November 2022 which means you are running FDS 6.8 or earlier. Does this case run for you with the current release?

**Reply by h7899** [reply]
Thank you

**Comment by rmcdermo** [comment]
This case completed successfully on our cluster with the latest source, but did give a few minimum density clip warnings around 0.098 kg/m3.  This does not seem problematic to me.  You could add a CLIP line to your input file that would reduce the MINIMUM_DENSITY to something like 0.01.  I suggest you rerun your case with this line and also using the latest version 6.9.1.
```
&CLIP MINIMUM_DENSITY=0.01/
```

**Reply by h7899** [reply]
Thank you
